bw2data/project.py

```diff
@@ -54,13 +62,83 @@ def __lt__(self, other):
         else:
             return self.name.lower() < other.name.lower()
 
+    def _get_base_directories(self):
```

Comment:
I don't quite understand how to read the diff here - the method `_get_base_directories` should already exist, but I don't see any lines removed? What am I missing?

Reply:
https://github.com/brightway-lca/brightway2-data/pull/174#discussion_r1644553443 see this earlier comment